dotcom-rendering/src/web/components/Dropdown.tsx

```diff
@@ -380,10 +380,9 @@ export const Dropdown = ({
 											)}
 										</a>
 
-										{l.notifications?.length &&
-											l.notifications.length > 0 && (
-												<RedDot />
-											)}
+										{!!l.notifications?.length && (
+											<RedDot />
+										)}
```

Comment:
Small refactor here, @guardian/tx-engineers. If the length is `0`, it would be falsy and therefore not render the dot.